src/styles/utils/_vars.scss

```diff
@@ -1 +1,6 @@
 $c-gray: #eee;
+$colorRed: #d12d35;
```

Comment:
Variable names should be consistent in naming convention. Consider changing '$colorRed' to '$c-red' for consistency.